content/applications/hr/time_off.rst

```diff
@@ -106,21 +78,76 @@ Allocation requests section
 - :guilabel:`Employee Requests`: select :guilabel:`Extra Days Requests Allowed` if the employee is
   able to request more time off than was allocated.
 
-  .. example::
-     For example, if ten (10) days are allocated to the employee for this particular type of time
-     off, and this option is enabled, the employee may submit a request for more than ten (10) days.
-
   If employees should **not** be able to make requests for more time off than what was allocated,
   select the :guilabel:`Not Allowed` option.
 
+  .. example::
+     Ten (10) days are allocated to the employee for this particular type of time off, and the
+     :guilabel:`Extra Days Requests Allowed` option is enabled. The employee wants to take a
+     vacation for twelve (12) days. They may submit a request for two (2) additional days since the
+     :guilabel:`Extra Days Requests Allowed` option is enabled.
+
+     It is important to note that requesting additional time off does not guarantee that time off is
+     granted.
+
```

Comment:
maybe we can just put this in a note or important admonition block, since "requesting additional time off does not guarantee that time off is granted" is probably true for all time off types